homeassistant/components/aseko_pool_live/binary_sensor.py

```diff
@@ -78,8 +79,8 @@ def __init__(
         """Initialize the unit binary sensor."""
         super().__init__(unit, coordinator)
         self.entity_description = entity_description
-        self._attr_name = f"{self._device_name} {entity_description.name}"
```

Comment:
I think this line should be removed only. Because binary sensor will fall back to the device class name, which fixes the problem as well

Reply:
That wouldn't help with the **Watter Flow** binary senor though.

What I can do than is to set the `_attr_translation_key` only if `translation_key` is set and set it only for the **Water Flow** and remove it for alarm/error if you would prefer.

Reply:
If you check the default implementation of the translation_key property:

```py
    @cached_property
    def translation_key(self) -> str | None:
        """Return the translation key to translate the entity's states."""
        if hasattr(self, "_attr_translation_key"):
            return self._attr_translation_key
        if hasattr(self, "entity_description"):
            return self.entity_description.translation_key
        return None
```

So no need to set it to shorthand attributes